behaviour/graql/language/define.feature

```diff
@@ -636,112 +598,61 @@ Feature: Graql Define Query
       | FAA |
 
 
-  Scenario: define abstract relation type creates a type
+  Scenario: define abstract relation type with no roleplayers creates a type
     Given graql define
       """
-      define membership sub relation, abstract, relates member;
+      define connection sub relation, abstract;
       """
     Given the integrity is validated
     When get answers of graql query
       """
-      match $x type membership; get;
+      match $x type connection; get;
       """
     Then concept identifiers are
       |     | check | value      |
-      | MEM | label | membership |
-    Then uniquely identify answer concepts
-      | x   |
-      | MEM |
-
-
-  Scenario: define concrete subtype of abstract relation creates child of its parent type
-    Given graql define
-      """
-      define
-      membership sub relation, abstract, relates member;
-      gym-membership sub membership, relates gym-with-members, relates gym-member as member;
-      """
-    Given the integrity is validated
-
-    When get answers of graql query
-      """
-      match $x sub membership; get;
-      """
-    Then concept identifiers are
-      |     | check | value          |
-      | MEM | label | membership     |
-      | GYM | label | gym-membership |
+      | CON | label | connection |
     Then uniquely identify answer concepts
       | x   |
-      | MEM |
-      | GYM |
+      | CON |
 
 
-  Scenario: define abstract subtype of abstract relation creates child of its parent type
+  Scenario: define relation type with multiple copies of the same 'relates' creates a relation type with all distinct roleplayers
```

Comment:
eh? this title no make sense to me

Reply:
I basically meant: _when you define a relation type with duplicate 'relates', you get a relation type that has all of the specified 'relates' as roleplayers, without any duplicates_.
It's hard to put it into a concise description.

Reply:
Renamed to `when defining a relation type, duplicate 'relates' are idempotent`.

Reply:
yas this makes sense :)